java/vector/src/main/java/org/apache/arrow/vector/BaseValueVector.java

```diff
@@ -104,7 +104,7 @@ protected ArrowBuf releaseBuffer(ArrowBuf buffer) {
 
   /* number of bytes for the validity buffer for the given valueCount */
   protected static int getValidityBufferSizeFromCount(final int valueCount) {
-    return (int) Math.ceil(valueCount / 8.0);
+    return valueCount % 8 > 0 ? valueCount / 8 + 1 : valueCount / 8;
```

Comment:
do you have benchmarks to show the difference here?

The [C++](https://github.com/apache/arrow/blob/master/cpp/src/arrow/util/bit-util.h) [implementation](https://github.com/apache/arrow/blob/43f2a31d3dd31cb2d5d6f0be72dba13a7a4e1e1f/cpp/src/arrow/util/bit-util.h#L110) has a different implementations to compute bytes for bits and round up to specific value without any branching.

Reply:
@emkornfield Thanks for your comments, do you mean _return (bits + 7) >> 3_ ? It seems more efficient.
I have just ran a test for this three implements use following code:
```
long start = System.nanoTime();
for (int i = 0; i < 10000; i++) {
      int res = expression;
 }
long end = System.nanoTime();
System.out.println(end- start);
```
The results are:
139502 (expression = (i + 7) >> 3)
237241 (expression = i % 8 > 0 ? i / 8 + 1 : i / 8)
3962019 (expression = (int)Math.ceil(i / 8.0))

I have replaced the expression with (valueCount + 7) >> 3, please have a look, thanks again.